src/views/createArticle/createArticle.scss

```diff
@@ -96,12 +95,14 @@ $black: #000;
 
 .btn {
   background-color: $white;
-  border: 2px solid $dark-col;
   border-radius: 8px;
   color: $dark-col;
   font-size: 20px;
   font-weight: bold;
-  padding: 8px 20px;
+  position: relative;
+  bottom: 16px;
+  // padding: 12px 20px;
```

Comment:
@Dubby20 please remove commented out code if not in use.

Reply:
Okay. Thanks